src/Admin.php

```diff
@@ -243,4 +246,17 @@ public static function admin_head() {
 EOD;
   }
 
+  /**
+   * Stops MCE editor setting attribute rel="noopener noreferrer"
+   * to links that opens in new window (target="_blank").
+   *
+   * @param array $mceInit MCD Editor configuration settings
+   *
+   * @return array
+   */
```

Comment:
1. The PHPDoc summary does not wrap.  You can use the current text as PHPDoc description and add a new summary (consider to copy the single-line inline comment from init()).

1. The name of the editor is TinyMCE.

1. The parameter description should be on an own line. See `get_user_option_user_settings()` in this file.

1. Instead of manually specifying the hook implementation's parameters and return value, you should refer to the documented filter instead by stating `@implements`.  See `hidden_meta_boxes()` in this file.

1. In case there is WP Core issue relating to this bug/behavior, it would be great to link to it with a `@see` directive at the end of the PHPDoc.